src/core/lib.zig

```diff
@@ -30,3 +30,5 @@ pub const MessageHeader = transaction.MessageHeader;
 pub const Transaction = transaction.Transaction;
 pub const VersionedTransaction = transaction.VersionedTransaction;
 pub const V0Message = transaction.V0Message;
+
+pub const Cluster = enum { mainnet, devnet, testnet, localnet };
```

Comment:
You don't need to change this now, but just some food for thought....

I think we may want to consolidate this somewhat with the ClusterType in accountsdb. We could keep the enum and union separate, but this enum could be the tag for that union. What do you think?

Maybe one issue is that there are some weird variants in the one in accountsdb. Development and Localhost seem redundant for example. I would say it's more important to support "custom" than either one of those, but maybe it's a good idea to explicitly have a localhost.

Reply:
yeah was thinking this as well but wasnt sure how - ill try the union tag approach